howto/troubleshoot/collect-info.md

```diff
@@ -0,0 +1,23 @@
+The guides in this section describe how to collect troubleshooting information
+from various parts of the Anbox Cloud stack. These guides should be used when
+facing or reporting issues with Anbox Cloud.
```

Comment:
```suggestion
from various parts of the Anbox Cloud stack. These guides should be used to
analyse or report issues with Anbox Cloud.
```
Reasoning for the change: "to face" and "to report" are so different that it is kinda confusing to see them combined like this. "Analyse" and "report" are a bit more similar.

Reply:
You're right, it's much nicer this way!